openmc/trigger.py

```diff
@@ -96,6 +97,20 @@ def add_score(self, score):
                   'it is not a string'.format(score)
             raise ValueError(msg)
 
+        # If this is a scattering moment, use generic moment order
```

Comment:
Is your comment about the code comment here I presume?
On Oct 20, 2015 3:15 PM, "sam" <notifications@github.com> wrote:

> In openmc/trigger.py
> <https://github.com/mit-crpg/openmc/pull/481#discussion_r42540427>:
>
> > @@ -96,6 +97,20 @@ def add_score(self, score):
> >                    'it is not a string'.format(score)
> >              raise ValueError(msg)
> >
> > +        # If this is a scattering moment, use generic moment order
>
> I think this could be a total or flux moment as well.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mit-crpg/openmc/pull/481/files#r42540427>.
>